### ISSUE TITLE
Display of potential rewards

### ISSUE THREAD

**Issue by NickShtefan** [GolosChain/golos#540]
Currently, the potential reward for the post is displayed in gbg. These data we get from the blockchain. However, in fact, the user receives a profit in 3 parts.
1. GOLOS POWER
2. GOLOS
3. GBG

This situation is misleading many users, it's difficult to understand what is realy happening.
It is necessary to update the data on the potential reward.

**Comment by VIM-Arcange** [comment]
IMHO, this looks like a UI issue and should be moved to https://github.com/GolosChain/tolstoy

**Comment by bitfag** [comment]
There is `'total_pending_payout_value': '78.229 GBG',` field which is misleading. Originally reward is in GOLOS and it is splitting at the payout.

I propose to change asset of `total_pending_payout_value` to GOLOS.

Optionally, would be nice to see what @NickShtefan says, for example new field which contains potential author payout:

```
{'pending_author_payout': { 'GOLOS': 1, 'GESTS': 1000, 'GBG': 1.43 }}
```

Also would be nice to see separate field for curators payout:

```
'pending_curator_payout': 'XXX GESTS'
```

**Comment by gropox** [comment]
Steemit (condencer) can already show the price in SP too. So it is more gui related feature and can be easily implemented.

![image](https://user-images.githubusercontent.com/26039472/38687898-108699c0-3e78-11e8-92b0-f247ce04abf7.png)

@bitfag , you forgot beficiaries. Otherwise could be missleading, especially golos.io plans to put it self to beneficieries.

**Comment by NickShtefan** [comment]
@bitfag Yes, this is exactly what is needed!

**Comment by bitfag** [comment]
@gropox sure, beneficiaries should be also counted. At the GUI level right now we can perform our own calculations to determine payout in GOLOS, then obtain sbd_print_rate, median price and estimate author payout in all assets. Also we can calculate actual curators reward percent. This is actually what upit does. Is this a best way? I doubt so. Why we need to reinvent blockchain mechanics at the application level? Are such calculations put huge demand on CPU cycles? Probably not.

So, my opinion is: we need to provide clear and easy to use data about all payout aspects via API.

**Comment by afalaleev** [comment]
Closed by #829
